rfc/0060-mod-menu.md

```diff
@@ -0,0 +1,73 @@
+
+## Summary
+
+This document outlines the process for creating an official Mod Menu for usage with Quilt alongside an accompanying team that would maintain it. This mod menu will act as a way to view loaded mods, change configs, read credits, and get links to contacts from within Minecraft.
+
+
+## Motivation
+   As of right now, we do not supply an official Mod Menu for Quilt. The current solution is TerraformerMC's Mod Menu for Fabric, and that does not fully support our additions (Full Quilt support, Quilt Config, quilt.mod.json spec improvements, etc.). Having a standardized Mod Menu would allow for support for these features. Furthermore, it would also be an officially endorsed mod, which would help us lead players to it, and we at The Quilt Project would be able to ensure its quality.
+
+
+## Explanation
+
+### Why are we doing this?
+Native Quilt Support, which lets us add additional features from things like quilt.mod.json or Quilt Config without issue.
+Standardization, allowing an official solution compared to Fabric's unofficial Mod Menu
+
+### Why not a fork of Terraformer's Mod Menu?
+The author of the Mod Menu himself, Prospector, said that ModMenu is "janky" - it is likely not worth the effort to clean up code AND integrate with other Quilt Standards. 
```

Comment:
I'll elaborate here. Mod Menu was created as one of the first Fabric mods and as time has gone on, it has increased in scope and capability. Admittedly, a lot of the code was rather haphazardly thrown together, and many remnants of the original code is visible, especially in the rendering part. The mod was also refactored to support multiple loaders more easily at some point, which also doesn't really make sense for a native mod. There are tons of undocumented features, such as localizable mod descriptions, which you would not know about unless you happened to find it in the code (I myself forgot I added the feature until I came across it in the code once).

I have advised that an official Quilt mod list screen be written from the ground up, however I will say that I am fairly happy with the UI that I have created as I put great effort into making the mod fit as seamlessly into vanilla as possible. I welcome anyone to take inspiration from or straight up rip-off my UI.